server/src/test/java/com/arcadedb/server/ws/WebSocketClientHelper.java

```diff
@@ -81,6 +81,12 @@ public void close() throws IOException {
     messageQueue.clear();
   }
 
+  public void breakConnection() throws IOException {
```

Comment:
The intent for this method was to simulate the case where the client "goes away" and doesn't send a close message. This will happen in the wild. :)

Reply:
Ack, I misread this change. All is well, please ignore me. :)

Reply:
NP. It still causes errors on CI. Locally all the tests pass, but not on CI. Weird. Can't find the reason. @arcade-player is helping on this.